diff --git a/src/es6/render/html_memory.es6 b/src/es6/render/html_memory.es6
index 9cc967d..55aff06 100644
--- a/src/es6/render/html_memory.es6
+++ b/src/es6/render/html_memory.es6
@@ -1,5 +1,7 @@
 /* eslint no-unused-vars: 0 */
-import {$, d3, uuid, jsplumb} from "nbtutor-deps";
+import {$, d3, uuid, jsPlumb} from "nbtutor-deps";
+
+const jsplumb = jsPlumb.jsPlumb;


 export class MemoryModelUI{
diff --git a/src/es6/vendor.es6 b/src/es6/vendor.es6
index b85efe4..56e4cb1 100644
--- a/src/es6/vendor.es6
+++ b/src/es6/vendor.es6
@@ -3,11 +3,12 @@ import _bind_polyfill from "phantomjs-polyfill";
 import $ from "jquery";
 import d3 from "d3";
 import uuid from "uuid";
-import jsplumb from "jsplumb";
+import jsPlumb from "jsplumb";
+

 export {
     $,
     d3,
     uuid,
-    jsplumb
+    jsPlumb
 };
